app/src/main/java/com/fish/monsters/features/safetyInformation/SafetyInformationScreen.kt

```diff
@@ -0,0 +1,146 @@
+    checkboxState: Boolean,
+    navigator: Navigator,
+    preferencesManager: PreferencesManager
+) {
+    var showAgain by remember { mutableStateOf(checkboxState) }
+
+    ScreenBox(title = stringResource(R.string.safety_information)) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(start = 16.dp, end = 16.dp, top = 30.dp),
+            verticalArrangement = Arrangement.spacedBy(20.dp)
+        ) {
+            Spacer(modifier = Modifier.height(60.dp))
+
+            Text(
+                text = stringResource(id = R.string.safety_warning),
+                style = MaterialTheme.typography.headlineMedium,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.fillMaxWidth(),
+                color = TextColorLight
+            )
+
+            SafetyInformationCard(
+                modifier = Modifier
+                    .fillMaxWidth()
+            ) {
+                Text(
+                    text = stringResource(id = R.string.safety_info_text),
+                    style = MaterialTheme.typography.bodyLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(all = 20.dp)
+                )
+            }
+
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 10.dp),
+                horizontalArrangement = Arrangement.Center,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+
+                Checkbox(
+                    checked = showAgain,
+                    onCheckedChange = {
+                        showAgain = it
+                        CoroutineScope(Dispatchers.IO).launch {
+                            preferencesManager.setShowSafetyScreen(it)
+                        }
+                    }
+                )
+                Spacer(Modifier.width(8.dp))
+                Text(text = stringResource(id = R.string.do_not_show_again))
+            }
+
+            Spacer(modifier = Modifier.weight(1f))
+
+            FishButton(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp),
+                onClick = {
+                    navigator.navigateTo(Screen.MainGameScreen)
+                }
+
+            ) {
+                Text(
+                    text = stringResource(id = R.string.start),
+                    style = MaterialTheme.typography.labelLarge
+                )
+            }
+        }
+    }
+}
+
+class MockPreferencesManager : PreferencesManager(null) {
+    override suspend fun setShowSafetyScreen(show: Boolean) {
+    }
+
+    override val showSafetyScreenFlow: Flow<Boolean> = flowOf(true)
+}
+
+class MockNavigator : Navigator() {
+
+    override fun navigateTo(screen: Screen) {
+    }
+
+    override fun navigateTo(screen: Screen, argument: String) {
+    }
+}
```

Comment:
Takie rzeczy się robi ewentualnie w testach, ale napewno nie tutaj, niestety głównie ze względów na wykorzystanie pamięci i wydajność. po za tym jest to po prostu śmietnik kodu, będziesz mógł to spokojnie usunąć po wprowadzeniu viewModel'u i wyniesieniu tego do koin'a. Po za tym to nawet teraz nie jest potrzebne bo równie dobrze możesz użyć w preview `Navigator()` oraz `PreferencesManager(null)` btw. context możesz uzyskać w każdej funkcji oznaczonej `@Composable` poprzez `LocalContext.current`